product-development/README.md

```diff
@@ -0,0 +1,83 @@
+
+The purpose of a single epic is _not_ to completely solve a business problem. Rather, completing an epic should likely achieve incremental progress at addressing a single problem, adding incremental user value as a result. In some cases, any realized user value may only appear in future epics. 
+
+After the epic is completed, the engineer should find the next epic to work on, either as a DRI owner, or as a collaborator. Engineers should self-organize and coordinate timelines ad hoc when picking epics. There are no stable teams, no timeboxed sprints, and no synced sprints across teams. Engineers should avoid working on multiple epics concurrently. 
+
+### Epic owner and epic states
+- An epic is in one of three states: `Unstarted`, `In Progress`, `Closed`. 
+- If an epic is in `In Progress` or `Closed`, it must have exactly one owner, who is the DRI.
+
+### DRI responsibilties and non-responsibilities as epic owner
+
+| What | Reponsibilities | Non-responsibilities | Why |
+| --- | --- | --- | --- |
+| Communication and status | The DRI is responsible for maintaining the epic state, description, and attached stories updated as the single source of truth of overall work status, at least on a daily basis.<br/><br/>The acceptance criteria sections are _especially critical_, including presenting the problem and final implementation in a company meeting.| The DRI is not responsible for communicating epic work status, beyond keeping the epic itself updated as the single source of truth. The DRI is not responsible for sharing work on Slack, meetings, or keeping Tilters otherwise updated. The DRI _can_ provide additional communication, but is not required to do so. Victor will help with broader general communication needs. | This encourages anyone who wants to learn about and collaborate on the epic to visit the epic itself (and it's attached stories) to see the latest updates, allowing the DRI to focus on actual work and leverage a single place for providing updates. |
+| Decision-making | The DRI is responsible for driving the epic to completion in a timely fashion within the target **_7 working days_**.<br/><br/>The DRI epic owner is responsible for reducing scope (deferring valuable work to future epics) if additional work is discovered. | The DRI is not responsible for seeking full consensus or permission from all Tilters.<br/><br/>The DRI is not responsible for waiting for a Tilter's response even if they asked for one, prior to making a decision. | We are biasing toward shipping in small increments quickly. So even if a sub-optimal change was shipped, this is preferable to _not_ shipping at all or shipping slowly, because we want to move quickly with small changes. The risk of doing something very bad is small since the scope is small, freeing up the DRI to have more confidence in their decision making, since the downside is low. If subsequent epics are needed to further tweak a shipped change, this is an indication of success, not failure. |
```

Comment:
The DRI is responsible for meeting the acceptance criteria, and so they may do this themselves by completing the tasks, or by asking someone else to do this. (i.e. project management of tasks and assignments). The acceptance criteria includes following up on customer conversations and writing docs and blog posts. Let me tweak the wording so that it's more intentional about getting immediate feedback from customers.

The acceptance criteria talks about linking to specific metrics that the change is supposed to improve. I think it would be very difficult to see metrics improve immediately shortly after a change is shipped. And I don't think it makes to have open epics that hang around for multiple weeks / months waiting on improvements. So I prefer if we could have a separate cycle / process to better measure how effective we are at evaluating success in the long run. In particular, I think this is something I could really help facilitate and organize.